migration/v0.39/apply/80-apply.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+# Add selector filters if covered by other snippets.
+# Example: "app!=something"
+declare -a skipped
+skipped=(
+)
```

Comment:
```suggestion
skipped=(
  "app!=trivy-operator"
)
```
And remove it from the v0.38's apply.